### DISCUSSION TITLE
how to enable system tray icon?

### DISCUSSION THREAD

**Discussion by Heus-Sueh** [sodaliterocks/sodalite#75] [Q&A] [answered]
:thinking:

**Comment by electricduck** [comment, marked as answer]
This is something explained in https://github.com/sodaliterocks/sodalite/issues/56#issuecomment-1386551314.

Indicators &mdash; "system tray icons" &mdash; are very much obsolete in the GTK world now, and the third-party tool we used to use in Fedora Linux 36-based versions of Sodalite no longer work in modern versions of Fedora Linux (they do however, still work in elementaryOS 7, but these will break eventually too). Most apps that have indicators will keep running in the background (I, myself, use a few: Discord, Nextcloud, Slack, Steam, and Telegram) though and should launch again correctly from the **Applications** menu when closed. GNOME has now introduced a panel for "background apps" in 44, and Pantheon will likely follow suite soon.

PRs attempting to fix the problem are welcome though if you're inclined to attempt to fix it &mdash; please, don't hesitate to ask in the [Telegram group for Sodalite](https://t.me/sodaliterocks) if you need help.

But, alas, indicator support is no longer a priority in Sodalite anymore. Sorry!
